airflow/operators/datetime.py

```diff
@@ -80,12 +80,12 @@ def choose_branch(self, context: Context) -> Union[str, Iterable[str]]:
             now = timezone.make_naive(context["logical_date"], self.dag.timezone)
         else:
             now = timezone.make_naive(timezone.utcnow(), self.dag.timezone)
```

Comment:
As far as I can tell the naive `now` is never used, so we can simply remove those `make_naive` calls?

Reply:
Crossed my mind - I just wasn't sure if there is a deeper meaning to it :)